DSharpPlus/Entities/DiscordMessageBuilder.cs

```diff
@@ -219,4 +219,16 @@ public async Task<DiscordMessage> ModifyAsync(DiscordMessage msg)
             return await msg.ModifyAsync(this).ConfigureAwait(false);
         }
     }
+
+    public struct DiscordFileBuilder
```

Comment:
This struct should be marked internal.

Reply:
Oh silly me, ill fix that today whenever i am able to actually sit

Reply:
Per discussions on Discord this is remaining public and other additions were made to the struct